src/main/java/fr/maxlego08/essentials/storage/database/repositeries/UserRepository.java

```diff
@@ -230,4 +230,11 @@ public void upsert(SunlightUser sunlightUser) {
             }
         });
     }
+
+    public void updateFrozen(UUID uuid, boolean frozen) {
```

Comment:
Same as the other query, you need to create a repository and execute the async queries